ingestion-beam/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Use maven image to build the jars that will be included via COPY later.
+FROM maven:3-jdk-8 AS build
+WORKDIR /app/ingestion-beam
+COPY pom.xml /app/pom.xml
+COPY ingestion-beam/pom.xml /app/ingestion-beam/pom.xml
+COPY checkstyle /app/checkstyle
+COPY ingestion-beam/checkstyle /app/ingestion-sink/checkstyle
+RUN mvn prepare-package
+COPY ingestion-core/src /app/ingestion-core/src
+COPY ingestion-beam/src /app/ingestion-beam/src
+RUN mvn package -Dmaven.test.skip=true
+
+# This is the base image for the final image we're building.
+FROM gcr.io/dataflow-templates-base/java8-template-launcher-base:20210120_RC00
```

Comment:
Can we add a comment here linking to the docs that list the current recommended base images?